app.js

```diff
@@ -11,63 +11,71 @@ const catchphraseInput = document.getElementById('catchphrase-input');
 const catchphraseButton = document.getElementById('catchphrase-button');
 
 // set state for how many times the user changes the head, middle, and bottom
+let headCount = 0;
+let middleCount = 0;
+let bottomCount = 0;
 // set state for all of the character's catchphrases
+let catchphrases = [];
 
 headDropdown.addEventListener('change', () => {
     // get the value of the head dropdown
-
+    const headChoice = headDropdown.value;
     // increment the head change count state
-    
+    headCount++;
     // update the dom for the head (use style.backgroundImage on the headEl div instead of trying to set the .src -- it's NOT an img tag!)
-
+    headEl.style.backgroundImage = `url(./assets/${headChoice}-head.png)`;
     // update the stats to show the new count (call displayStats() to do this work)
+    displayStats();
 });
 
-
 middleDropdown.addEventListener('change', () => {
     // get the value of the middle dropdown
-
+    const middleChoice = middleDropdown.value;
     // increment the middle change count state
-    
+    middleCount++;
     // update the dom for the middle (NOTE: use style.backgroundImage on the middleEl div instead of trying to set the .src -- it's NOT an img tag!)
-
+    middleEl.style.backgroundImage = `url(./assets/${middleChoice}-middle.png)`;
     // update the stats to show the new count (call displayStats() to do this work)
+    displayStats();
 });
 
-
 bottomDropdown.addEventListener('change', () => {
     // get the value of the bottom dropdown
-
+    const bottomChoice = bottomDropdown.value;
     // increment the bottom change count state
-    
+    bottomCount++;
     // update the dom for the bottom (NOTE use style.backgroundImage on the bottomEl div instead of trying to set the .src -- it's NOT an img tag!)
-
+    bottomEl.style.backgroundImage = `url(./assets/${bottomChoice}-pants.png)`;
     // update the stats to show the new count (call displayStats() to do this work)
+    displayStats();
 });
 
 catchphraseButton.addEventListener('click', () => {
     // get the value of the catchphrase input
-    
     // push the new catchphrase to the catchphrase array in state
-
     // clear out the form input's value so it's empty to the user
```

Comment:
We're missing this step here. If we set the input's value to an empty string, that will clear the input so that we don't have the text sticking around.